js_modules/dagit/packages/core/src/ui/Tabs.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import {ColorsWIP} from './Colors';
+import {IconWrapper} from './Icon';
+import {FontFamily} from './styles';
+
+interface TabsProps {
+  children: Array<React.ReactElement<TabProps>>;
+  selectedTabId?: string;
+  onChange?: (selectedTabId: string) => void;
+  small?: boolean;
```

Comment:
Minor, but I kind of prefer `size: 'small' | 'large';` with a default.

I always find it confusing when some components have `small: boolean` and others have `large: boolean`.